Moving astro vs jsx comparison (formerly chart) and resulting fallout (#1188)
withastro/docs: src/pages/en/guides/troubleshooting.md

### My component is not rendering

> First, check to see that you have **imported the component** in your [`.astro` component script](/en/core-concepts/astro-components/#the-component-script) or [`.mxd`](/en/guides/markdown-content/#using-components-in-mdx) file.

Reviewer: Typo 😅
```suggestion
First, check to see that you have **imported the component** in your [`.astro` component script](/en/core-concepts/astro-components/#the-component-script) or [`.mdx`](/en/guides/markdown-content/#using-components-in-mdx) file.
```

> First, check to see that you have **imported the component** in your [`.astro` component script](/en/core-concepts/astro-components/#the-component-script) or [`.mdx`](/en/guides/markdown-content/#using-components-in-mdx) file.

Reviewer 2: "file" feels like it belongs inside the link here?

```suggestion
First, check to see that you have **imported the component** in your [`.astro` component script](/en/core-concepts/astro-components/#the-component-script) or [`.mdx` file](/en/guides/markdown-content/#using-components-in-mdx).
```

Then check your import statement:

- Is your import linking to the wrong place? (Check your import path.)

> - Does your import have the same name as the imported component? (Check your component name and that it [follows the `.astro` syntax](/en/core-concepts/components/#astro-vs-jsx).)

Author: ```suggestion
- Does your import have the same name as the imported component? (Check your component name and that it [follows the `.astro` syntax](/en/core-concepts/astro-components/#astro-vs-jsx).)
```

> - Does your import have the same name as the imported component? (Check your component name and that it [follows the `.astro` syntax](/en/core-concepts/astro-components/#astro-vs-jsx).)

Author: ```suggestion
- Does your import have the same name as the imported component? (Check your component name and that it [follows the `.astro` syntax](/en/core-concepts/astro-components/#differences-between-astro-and-jsx).)
```
